pfcpiface/parse-far.go

```diff
@@ -141,6 +147,18 @@ func (f *far) parseFAR(farIE *ie.IE, fseid uint64, upf *upf, op operation) error
 			} else if f.dstIntf == ie.DstInterfaceCore {
 				f.tunnelIP4Src = ip2int(upf.coreIP)
 			}
+		case ie.PFCPSMReqFlags:
+			fields = Set(fields, FwdIEPfcpSMReqFlags)
```

Comment:
Is this required? We don't seem to use the fields anywhere

Reply:
Lets keep the set fields for now. Will remove later if not used..